tests/models/votable_test.py

```diff
@@ -42,3 +42,51 @@ class TestModel(BaseModel):
     assert model.type == VOTablePrimitive.double
     assert model.type.pack(13.71) == struct.pack(">d", 13.71)
     assert model.model_dump(mode="json") == {"type": "double"}
+
+
+def test_votable_primitive_unicode_char() -> None:
```

Comment:
These tests are just of the simple pack in the model, but that's only used if there's a single character, so this doesn't test any of the exciting pieces. I think you need some new tests in `tests/storage/votable_test.py` (and they should include Unicode strings that require surrogate pairs, particularly ones where the truncation falls inside a character).